CONTRIBUTING.md

```diff
@@ -61,8 +61,16 @@ If you are proposing a feature:
 
 ## Documentation
 
-The latest API documentation is usually available
-[here](https://airflow.incubator.apache.org/). To generate a local version,
+The Airflow documentation is located at:
+
+- <https://airflow.apache.org> (points to
```

Comment:
This is not true as I said in the comments https://airflow.apache.org hosts the docs for stable version on Apache website. RTD stores versioned docs including stable but that is on RTD servers not apache.

Reply:
@kaxil Thank you for clarifying.  I was operating under the assumption that `airflow.apache.org` was set as a `CNAME` to `airflow.readthedocs.io` using the domain setting on Read the Docs (https://readthedocs.org/dashboard/airflow/domains/).  Is this something that's been considered?

Is the longterm plan to keep the separate Apache site?  For instance, suppose we have 2 LTS releases like 1.10 and 2.0, would the Apache site then serve both versions with the version dropdown?

The reference example I'm thinking of is how Django hosts all versions on one site - https://docs.djangoproject.com/ (try bottom right version button).

Reply:
I think for now the plan is to stick with apache for stable docs and not adding anything new or redirecting it to readthedocs (atleast for time being).

Reply:
If we want to have versions on something we host there is https://robpol86.github.io/sphinxcontrib-versioning/ -- I tried it but I looked at using it for some internal docs (I haven't yet though)